packages/git/src/node/init/git-init.ts

```diff
@@ -25,27 +30,63 @@ export interface GitInit extends Disposable {
 
     /**
      * Called before `Git` is ready to be used in Theia. Git operations cannot be executed before the returning promise is not resolved or rejected.
-     *
-     * This implementation does nothing at all.
      */
     init(): Promise<void>;
 
 }
 
 /**
- * The default initializer. It is used in the browser. Does nothing at all.
+ * The default initializer. It is used in the browser.
+ *
+ * Configures the Git extension to use the Git executable from the `PATH`.
  */
 @injectable()
 export class DefaultGitInit implements GitInit {
 
     protected readonly toDispose = new DisposableCollection();
 
+    @inject(ILogger)
+    protected readonly logger: ILogger;
+
+    @inject(MessageService)
+    protected readonly messages: MessageService;
+
     async init(): Promise<void> {
-        // NOOP
+        const { env } = process;
+        if (typeof env.LOCAL_GIT_DIRECTORY !== 'undefined' || typeof env.GIT_EXEC_PATH !== 'undefined') {
+            await this.handleExternalNotFound('Cannot use Git from the PATH when the LOCAL_GIT_DIRECTORY or the GIT_EXEC_PATH environment variables are set.');
+        } else {
+            try {
+                const { execPath, path, version } = await findGit();
+                if (!!execPath && !!path && !!version) {
+                    // https://github.com/desktop/dugite/issues/111#issuecomment-323222834
+                    // Instead of the executable path, we need the root directory of Git.
+                    const dir = dirname(dirname(path));
+                    const [execPathOk, pathOk, dirOk] = await Promise.all([pathExists(execPath), pathExists(path), pathExists(dir)]);
+                    if (execPathOk && pathOk && dirOk) {
+                        process.env.LOCAL_GIT_DIRECTORY = dir;
+                        process.env.GIT_EXEC_PATH = execPath;
+                        this.logger.info(`Using Git [${version}] from the PATH. (${path})`);
+                        return;
+                    }
+                }
+                await this.handleExternalNotFound();
+            } catch (err) {
+                await this.handleExternalNotFound(err);
+            }
+        }
     }
 
     dispose(): void {
         this.toDispose.dispose();
     }
 
+    // tslint:disable-next-line:no-any
+    protected async handleExternalNotFound(err?: any): Promise<void> {
+        if (err) {
+            this.logger.error(err);
+        }
+        this.messages.info('Could not find Git on the PATH. Falling back to the embedded Git.');
```

Comment:
It won't fall back. I know it was my code :) Please remove `Falling back to the embedded Git.`.

Reply:
Done